layers/+completion/ivy/packages.el

```diff
@@ -184,7 +183,8 @@
       (ido-mode -1))))
 
 (defun ivy/init-ivy-hydra ()
-  (use-package ivy-hydra))
+  (use-package ivy-hydra)
+  (define-key hydra-ivy/keymap [escape] 'hydra-ivy/keyboard-escape-quit-and-exit))
```

Comment:
Ivy autoloads `ivy-hydra` when `hydra-ivy/body` is called, so it should be safe to use `with-eval-after-load` instead of `use-package`. Please change and test.